appservice/src/connectToGitHub.ts

```diff
@@ -108,13 +92,12 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient):
     }
 }
 
-async function showGitHubAuthPrompt(): Promise<void> {
-    const learnMore: string = localize('learnMore', 'Learn More');
-    const setupGithub: string = localize('setupGithub', 'You must give Azure access to your GitHub account.');
-    const input: string | undefined = await vscode.window.showErrorMessage(setupGithub, learnMore);
-    if (input === learnMore) {
-        // tslint:disable-next-line:no-unsafe-any
-        opn('https://aka.ms/B7g6sw');
+async function showGitHubAuthPrompt(node: AzureTreeItem): Promise<void> {
+    const invalidToken: string = localize('tokenExpired', 'Azure\'s GitHub token in invalid.  Authorize in the "Deployment Center"');
+    const goToPortal: string = localize('goToPortal', 'Go to Portal');
```

Comment:
It's still not the best experience, but there is a Github pane.  Ideally, we don't want to direct people to the Portal so we don't lose users, but I figure it's better to be authorized in the portal than not at all.

![image](https://user-images.githubusercontent.com/5290572/48369184-abea9700-e66a-11e8-8974-99548ae1ba9d.png)

Reply:
I was kind of just wondering if we wanted the "Open in Portal" _and_ the "Learn more" button

Reply:
Do you want to link this? https://docs.microsoft.com/en-us/azure/app-service/app-service-continuous-deployment#deploy-continuously-from-github

Reply:
Either that or you write your own doc. Either way you can always change it since it's an aka.ms link

If you add both - definitely add telemetry. That way we can remove one of the buttons if no one is using it